src/ui/common/src/utils/integrations.ts

```diff
@@ -351,11 +351,14 @@ export const SupportedIntegrations: ServiceInfoMap = {
     activated: true,
     category: 'data',
   },
+  // TODO: enable this once we release conda
+  /*
   ['Conda']: {
     logo: 'https://aqueduct-public-assets-bucket.s3.us-east-2.amazonaws.com/webapp/pages/integrations/conda.png',
     activated: true,
```

Comment:
@cw75, can't we just set `activated` to `false` here?

Reply:
If we set it to `false`, I think the UI will still show the Conda box but in a greyed-out state. If we're ok with this we can do that.

Reply:
Perhaps we can try to add a hidden flag in the future, but that would effectively do the same thing here.